index.md

```diff
@@ -1,87 +1,43 @@
 
-Next Meeting/Event <!-- You should keep this section as it will populate your meetup events -->
----------------------
-{% include chapter_events.html group=page.meetup-group %}
+Referrals to this website or to individual meetings to colleagues or acquaintances are welcome.
 
-<!-- You should delete this comment
+## Upcoming Events
 
-Standard Chapter Page Template
-This is an example of a Project or Chapter page.
-Please change these items to indicate the actual information you wish to present. In addition to this information, the 'front-matter' above the text should be modified to reflect your actual information.  An explanation of each of the front-matter items is below:
+Check out our [Upcoming Events](https://www.meetup.com/de-DE/OWASP-Stuttgart/events/)
 
-{front matter for this file}
+## Information about Stuttgart Chapter Events
 
-```
-- layout: This is the layout used by project and chapter pages.  You should leave this value as col-sidebar
-- title: This is the title of your project or chapter page, usually the name.  For example, OWASP Zed Attack Proxy or OWASP Baltimore
-- tags: This is a space-delimited list of tags you associate with your project or chapter.  If you are using tabs, at least one of these tags should be unique in order to be used in the tabs files (an example tab is included in this repo) 
-- region: This is the region you are in according to our data
-```
+### When?
 
-{copy for this file (index.md)}
-Replace the text above the commented area with your information in the format below:
-```
-## Welcome
-Include some information here about your chapter
+We will be running quarterly chapter meetups with the option to have additional events such as hands-on workshops. 
 
-## Participation
-The Open Worldwide Application Security Project (OWASP) is a nonprofit foundation that works to improve the security of software. All of our projects ,tools, documents, forums, and chapters are free and open to anyone interested in improving application security. 
+The dates will be picked randomly to avoid recurring conflicts with other events (meetups, private obligations, holiday etc.). Typically, we chose a day from Tuesday to Thursday in the week, since our experience in organizing events is, that Mondays and Fridays have a very high no-show rate 😬
 
-Chapters are led by local leaders in accordance with the [Chapter Leader Handbook](/www-policy/rules-of-procedure/chapter-handbook). Financial contributions should only be made online using the authorized online donation button. To be a SPEAKER at ANY OWASP Chapter in the world simply review the [speaker agreement](/www-policy/speaker-agreement) and then contact the local chapter leader with details of what OWASP Project, independent research, or related software security topic you would like to present.
+Typically, the event starts at 18:00 with arriving and socialising. Starting at 18:30 with the talk of our guest speaker or whatever we've planned for the particular event.
```

Comment:
Events usually start at 18:00 with guests arriving and socialising. By 18:30, we proceed with the talk from our guest speaker or dive into the planned activities for the evening.